### PR TITLE
Feat: Added product page

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added product page:
1) filter sidebar.
     *Filter by shoe size.
     *Filter by Brands.
     *Filter by Colors.
     *Filter by price.
     *Ratings.
2) product card with description. 
chrome-capture

### CODE REVIEW

File: css/product.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/* .nav-logo i {
+  cursor: pointer;
+} */
+.filter-bar {
+  z-index: 1;
+  top: 0;
+  position: fixed;
+  left: 0;
+  width: 300px;
```
Comment (ramakrishnach14) [inline css/product.css:9]: would be better if relative units

File: css/product.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  padding: 0.7rem;
+  border: none;
+  background-color: var(--header-backgroundColor);
+  color: var(--header-Color);
+  cursor: pointer;
+}
+.btn:hover {
+  opacity: 0.6;
+}
+@media screen and (max-width: 780px) {
```
Comment (ramakrishnach14) [inline css/product.css:111]: better to maintain different screen sizes in variables

**Review by ramakrishnach14 — COMMENTED** [review]
looks Good

**Review by vijaytembugade — APPROVED** [review]
Working fine. Just see Shop now button on home screen, you can overrite the anchor tag default style.

File: css/product.css (hunk 1 above)
Comment (abhijeetekad) [reply css/product.css:9]: Ok

**Pull request merged**